product_docs/docs/edbcloud/beta/using_cluster/connecting_your_cluster/01_connecting_from_azure/index.mdx

```diff
@@ -0,0 +1,81 @@
+
+**Pros**
+ - Service provider's and consumer's virtual networks can be in different regions or subscriptions.
+ - Single set up can be shared with all consumers. 
+ - No risk of IPs overlapping. 
+ - Traffic passes securely through the Microsoft backbone network and not the internet.                                              
+ - Most Azure homebrew services (such as CosmosDB) have built-in 
+  [private link resources](https://docs.microsoft.com/en-us/azure/private-link/private-endpoint-overview#private-link-resource) that you can connect to via a private endpoint.
+
+**Cons**
+ - EDB Cloud virtual network can not initiate traffic to your application.
+ - There is an associated cost. See [Azure Private Link pricing](https://azure.microsoft.com/en-us/pricing/details/private-link/#pricing).
+
+See the [Private Endpoint Example](01_private_endpoint) for the steps for connecting using this method.
+
+## Virtual Network Peering
+
+Virtual network peering connects two Azure virtual networks. Once peered, the two virtual networks act as one with respect to connectivity.
+
+**Pros**
+- Service provider's and consumer's virtual networks can be in different regions, subscriptions, or tenants. 
+- Simple and easy to set up.
+- Traffic passes securely through the Microsoft backbone network and not the internet.  
+
+**Cons**
+- There is an associated cost. See [pricing for virtual network peering](https://azure.microsoft.com/en-us/pricing/details/virtual-network/#pricing)  for details.
+- The IP ranges of two peered virtual networks can not overlap.  
+
+See the [Viritual Network Peering Example](02_virtual_network_peering) for the steps for connecting using this method.
+
+## Azure VNet-VNet Connection
+
+A VPN gateway is used to send encrypted traffic between Azure virtual networks. You can
+connect your EDB Cloud virtual network and your client VM virtual network by using the VNet-to-VNet VPN gateway connection.
+
+**Pros**
+- Service provider's and consumer's virtual networks can belong to different regions, subscriptions, or tenants. 
+- Traffic passes through Microsoft backbone network and not the internet. 
+- PG service domain name is directly accessible without a NAT. 
+- VNets from different subscriptions do not need to be associated with the same Active Directory tenant.
+
+**Cons**
+- Limited bandwidth, see [virtual network gateway planning table](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-about-vpngateways#planningtable). 
+- Set up is complicated. 
+- There is an associated cost, see [virtual network gateway planning table](https://docs.microsoft.com/en-us/azure/vpn-gateway/vpn-gateway-about-vpngateways#planningtable).
+
+See the [VNet-VNet Example](03_vnet_vnet) for the steps for connecting using this method.
+
+
+
+## Draft of alternative consolidated format for the Pros and Cons lists
```

Comment:
IMO we should stick with the bulleted list - the pros and cons don't line up that cleanly.